src/main/java/mate/academy/app/controller/ReviewsController.java

```diff
@@ -11,13 +26,45 @@
 @RequestMapping("/reviews")
 public class ReviewsController {
```

Comment:
let's use @AllArgsConstructor(or @RequiredArgsConstructor) everywhere to keep your code consistent.